al_log_filter.js

```diff
@@ -38,10 +38,18 @@ var initJsonFilter = function(filter) {
  *  @NOTE: If json filter  is bad then 'messages' is returned unfiltered.
  */
 
-var filterJson = function(messages, filter) {
+var filterJson = function (messages, filter) {
     const filterJ = initJsonFilter(filter);
+    let result = [];
     if (filterJ) {
-        return where(messages, filterJ);
+        if (Array.isArray(filterJ)) {
+            filterJ.forEach(function (e) {
+                result = result.concat(lodashFilter(messages, e));
```

Comment:
it looks like if a message satisfies multiple filters then it will be added multiple time into `result`. Could you please double check this?